docs/reference/certificates.md

```diff
@@ -1,85 +1,83 @@
-5. Client certificate for the API server to talk to etcd
-6. Client certificate/kubeconfig for the controller manager to talk to the API server
-7. Client certificate/kubeconfig for the scheduler to talk to the API server.
-8. Client and server certificates for the [front-proxy][proxy]
+Kubernetes requires PKI for the following operations:
 
+* Client certificates for the kubelet to authenticate to the API server
+* Server certificate for the API server endpoint
+* Client certificates for administrators of the cluster to authenticate to the API server
+* Client certificates for the API server to talk to the kubelets
+* Client certificate for the API server to talk to etcd
+* Client certificate/kubeconfig for the controller manager to talk to the API server
+* Client certificate/kubeconfig for the scheduler to talk to the API server.
+* Client and server certificates for the [front-proxy][proxy]
 
-In addition, etcd uses mutual TLS for authenticating clients and peers.
+etcd also implements mutual TLS to authenticate clients and peers.
 
-## Where to find them
+## Where certificates are stored
 
 Kubernetes stores certificates by default in `/etc/kubernetes/pki`. All paths in this documentation are relative to that directory.
 
-## Certificate configuration
+## Configure certificates manually
 
-There are multiple levels of control for kubernetes certificates. 
-At the basic level, a tool like [kubeadm][kubeadm] can generate all required certificates.
-If that isn't desireable, a single root CA, controlled by an adminstrator, can create multiple _intermediate CAs_, and delegate all further creation to Kubernetes itself. 
+If you don't want [kubeadm][kubeadm] to generate the required certificates, you can create them in either of the following ways.
 
+### Single root CA
 
-The CAs needed for this to work are:
+You can create a single root CA, controlled by an adminstrator. This root CA can then create multiple intermediate CAs, and delegate all further creation to Kubernetes itself. 
+
+Required CAs:
 
 | path                   | Default CN                | description                      |
 |------------------------|---------------------------|----------------------------------|
-| ca.crt,key             | kubernetes-ca             | the kubernetes general CA        |
-| etcd/ca.crt,key        | kubernetes-front-proxy-ca | For all ETCD-related functions   |
-| front-proxy-ca.crt,key | etcd-ca                   | for the [front-end proxy][proxy] |
-
-If you don't wish to copy these private keys to your API servers, you can generate all certificates yourself. 
-In that case, you will also need to make all of these:
+| ca.crt,key             | kubernetes-ca             | Kubernetes general CA            |
+| etcd/ca.crt,key        | etcd-ca                   | For all etcd-related functions   |
+| front-proxy-ca.crt,key | kubernetes-front-proxy-ca | For the [front-end proxy][proxy] |
```

Comment:
It looked as though this and the previous line somehow got reversed in the version that I edited. If I've misunderstood something obscure, my apologies (but if I've misunderstood, we should explain a bit more too).